.github/workflows/xpu-max1100.yml

```diff
@@ -6,7 +6,7 @@ on:
     - cron: "0 0 * * *"
   pull_request:
     paths:
-      - ".github/workflows/xpu.yml"
+      - ".github/workflows/xpu-max1100.yml"
```

Comment:
FYI @Liangliang-Ma this is why it didn't run on your PR originally but it is now.